django/test/testcases.py

```diff
@@ -1085,11 +1085,7 @@ def _fixture_setup(self):
                     apps.set_available_apps(self.available_apps)
 
             if self.fixtures:
-                # We have to use this slightly awkward syntax due to the fact
-                # that we're using *args and **kwargs together.
```

Comment:
Well, I traced this back to 01adbb55e66... Things have clearly improved after 16 years! 😂